src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import './index.scss';
```

Comment:
since this .scss file is empty, its good practice to just delete them and only re-create them if you need them later.